test/distributed/_tensor/test_dtensor_compile.py

```diff
@@ -234,8 +234,8 @@ def fn(x):
                 requires_grad=x.requires_grad,
             )
 
-        out = fn(x)
-        out2 = torch.compile(fn, backend="eager")(x)
+        fn(x)
+        torch.compile(fn, backend="eager")(x)
```

Comment:
cc @wanchaol is this supposed to check the output match? Or just that both run properly?